packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2581,6 +2581,9 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     super.dispose();
   }
 
+  /// Delegate triggered when we remove the bottom insets
```

Comment:
avoid "we" in doc comments, see style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require